packages/@sfx/sayt-plugin/test/unit/browser/sayt-plugin.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect, sinon, spy, stub } from '../../utils';
+import SaytPlugin from '../../../src/sayt-plugin';
+import * as SaytPackage from 'sayt';
+
+describe('SaytPlugin', () => {
+  let saytPlugin: any;
+
+  beforeEach(() => {
+    saytPlugin = new SaytPlugin();
+  });
+
+  describe('metadata getter', () => {
```

Comment:
We probably also want to add a test that ensures that the plugin name is the one that we expect. This makes it so that if someone on the team accidentally changes the name that the test suite will fail and inform them.